backend/core/src/main/java/org/sonarsource/sonarlint/core/repository/config/ConfigurationRepository.java

```diff
@@ -62,6 +63,19 @@ public ConfigurationScopeWithBinding remove(String idToRemove) {
     return removedScope == null ? null : new ConfigurationScopeWithBinding(removedScope, removeBindingConfiguration);
   }
 
+  public Map<String, BindingConfiguration> removeBindingForConnection(String connectionId) {
+    var removedBindingByConfigScope = new HashMap<String, BindingConfiguration>();
+    var configScopesToUnbind =
```

Comment:
Small suggestion: `configScopesToUnbind` -> `configScopeIdsToUnbind`

Reply:
Addressed in [d9d5ad2](https://github.com/SonarSource/sonarlint-core/pull/982/commits/d9d5ad261521e1be145998b0fa925f68b0981f2a)